docs/adrs/003-Retry_endpoint_design.md

```diff
@@ -35,26 +35,26 @@ The endpoint takes a job or list of job ids and then attempts to resubmit them t
 * EE2 looks up the job versions and parameters, and then submits the job to be retried, incrementing the `retry_count`
   of the job being retried, and the newly launched job gains a pointer to the `_PARENT_RETRY_JOB_ID`
 * The job is submitted and upon successful submission, notifies the `retry_parent` and notifies the `parent_job_id` that a new `child_job` has been added
+* Jobs submitted by an administrator with a ResourceRequirements use resource requirements from the catalog / ee2 config exclusively on retry
 
-
-### Batch Behavior
-* If a job has the attribute of `batch_job=True` the retry will fail, since there is no method to re-run. This is a bug, as it doesn't fail gracefully.
-* If a job has the attribute of `batch_job=True`, but is actually a child job, the parent will be notified of this new retried job
-* Multiple in-flight retries are allowed.
+### Batch Jobs Behavior
+* Adds child_job_id to parent_job_id.child_job_ids
```

Comment:
If I'm reading this right, the parent job just gets a new child job id for the retried job. I may not have followed, is there a way to get the original job from the retried job id?

Reply:
Yes, the original job is available via `retry_parent` field

Reply:
can you ensure that you quote field names consistently throughout the ADR?

Reply:
Alright, I think I got them all